Dockerfile

```diff
@@ -44,16 +44,19 @@ ARG TARGETARCH
 RUN --mount=type=cache,target=/var/cache/apt \
     apt-get update && \
         # Install dependencies to add additional repos
-        apt-get install -y --no-install-recommends ca-certificates curl && \
-        # FIXME Node 18 actually shouldn't be necessary in Community, but we assume its presence in lots of tests
-        curl -sL https://deb.nodesource.com/setup_18.x | bash - && \
-        apt-get update && \
+        apt-get install -y --no-install-recommends ca-certificates curl gnupg && \
         apt-get install -y --no-install-recommends \
             # Runtime packages (groff-base is necessary for AWS CLI help)
-            git make openssl tar pixz zip unzip groff-base iputils-ping nss-passwords procps \
-            # FIXME Node 18 actually shouldn't be necessary in Community, but we assume its presence in lots of tests
-            nodejs
+            git make openssl tar pixz zip unzip groff-base iputils-ping nss-passwords procps
```

Comment:
I think this could be a single `apt-get install`. It was only separated because `gnupg` and `curl` are used for the `nodejs` installation and have therefore been installed separately before.

Reply:
Thanks, makes sense :+1: